### ISSUE TITLE
Reloading a batch seems to have a bug in it

### ISSUE THREAD

**Issue by Sherlock21** [FreeUKGen/MyopicVicar#1008]

**Comment by Sherlock21** [comment]
I had a file: WRY/SEDBU2.csv which had about 500 records, form months ago.
Then yesterday, I uploaded a replacement file that had more records in it - 954 including the header rows -
this replace mane had 3 data errors showing - so I fixed me PC version of the file and at the same timeI noticed that I had forgotten to change the Case of some fields, so I fixed that too before uploading a Replace again.
The system email advised that it had been processed OK
This nothing,I checked to see if the Errors Counter had reset to zero, and it has and it hasn't - i.e. there are NO errors in the file if I click on Show errors. But the number of errors is still showing as 3 not he  File Information page!
I downloaded the file, to check it. and the downloaded file has  1255 records including the header rows.

So what's not getting replaces when it should? 
Theres a discrepancy of 301 records here. is it all to do with me changing Case of Surnames? or what.

**Comment by edickens** [comment]
EricB - Do you think that you may have uploaded the "Change the
    case" before the previous upload had completed?  Did you get the
    email before you uploaded again?  This might cause a mix up.  E
    
      On 12/11/2016 09:35, Sherlock21 wrote:
    
    
      I had a file: WRY/SEDBU2.csv which had about 500 records, form
        months ago.
        Then yesterday, I uploaded a replacement file that had more
        records in it - 954 including the header rows -
        this replace mane had 3 data errors showing - so I fixed me PC
        version of the file and at the same timeI noticed that I had
        forgotten to change the Case of some fields, so I fixed that too
        before uploading a Replace again.
        The system email advised that it had been processed OK
        This nothing,I checked to see if the Errors Counter had reset to
        zero, and it has and it hasn't - i.e. there are NO errors in the
        file if I click on Show errors. But the number of errors is
        still showing as 3 not he File Information page!
        I downloaded the file, to check it. and the downloaded file has
        1255 records including the header rows.
      So what's not getting replaces when it should? 
        Theres a discrepancy of 301 records here. is it all to do with
        me changing Case of Surnames? or what.
      —
        You are receiving this because you are subscribed to this
        thread.
        Reply to this email directly, view
          it on GitHub, or mute
          the thread.
      
        
          
          
        
        
      
      {"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/FreeUKGen/MyopicVicar","title":"FreeUKGen/MyopicVicar","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/FreeUKGen/MyopicVicar"}},"updates":{"snippets":[{"icon":"PERSON","message":"@Sherlock21 in #1008: I had a file: WRY/SEDBU2.csv which had about 500 records, form months ago.\nThen yesterday, I uploaded a replacement file that had more records in it - 954 including the header rows -\nthis replace mane had 3 data errors showing - so I fixed me PC version of the file and at the same timeI noticed that I had forgotten to change the Case of some fields, so I fixed that too before uploading a Replace again.\nThe system email advised that it had been processed OK\nThis nothing,I checked to see if the Errors Counter had reset to zero, and it has and it hasn't - i.e. there are NO errors in the file if I click on Show errors. But the number of errors is still showing as 3 not he  File Information page!\nI downloaded the file, to check it. and the downloaded file has  1255 records including the header rows.\n\nSo what's not getting replaces when it should? \nTheres a discrepancy of 301 records here. is it all to do with me changing Case of Surnames? or what."}],"action":{"name":"View Issue","url":"https://github.com/FreeUKGen/MyopicVicar/issues/1008#issuecomment-260112038"}}}

**Comment by Sherlock21** [comment]
The original file had been uploaded long before the recent outage.

Then my first Reload was well completed, ( as an asap) and I had had the email for may be half an hour, then I went to leek at the 3 date errors.
Explored online (in my batches) to see  what they were. And decided th correct my file and reload it.

It Was then I noticed there were some records at the end of my file that were all in Lower case ( because I had forgotten to  set  it right)

So when I had fixed that, and created a new .csv file to upload, must have been 10 minutes or more.

Then Reloaded this corrected file into my Batches. it ran as asap, and I got the email: all Done - withing a minute or two.

Now - I know there are some indexes that don’t update until overnight, so I awaited to this morning expecting all to be sorted by then - but the file Information page still showed 3 errors - which I woudl have expected by then to have sorted its self out. 
It was then I downloaded a copy of the file, so I could try and find them. That’s when I found there were far more rows than there should have been, and just wondered if it was the case change that did not count as a change and created a new row?

Its too difficule to find that out by comparing the downloaded fuel with my original, because the rows of the downloaded file are all over the place unfortunately.


I have not noticed that sort of result in the past, when I have uploaded a replace so soon after the first load.
EB
